From 90bf495b0002d84f2bcabf094626b4d35041ef92 Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Mon, 15 Jan 2018 00:43:35 +0100
Subject: [PATCH 2/2] Set up a conflict with standard ROS pkg and other OSRF
 wrappers

---
 debian/control.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control.em b/debian/control.em
index 9b99e73..e46e6c0 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -9,6 +9,7 @@ Standards-Version: 3.9.2
 Package: @(Package.replace('gazebo-','gazebo8-'))
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Conflicts: @(Package), @(Package.replace('gazebo-','gazebo6-')), @(Package.replace('gazebo-','gazebo7-'))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.7.4

