From 66f0fd5b9258108ddcf53c21d3d4887953b0fff1 Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed, 22 Mar 2017 19:47:27 +0100
Subject: [PATCH 2/2] Set up a conflict with standard ROS pkg

---
 debian/control.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control.em b/debian/control.em
index 9b99e73..2e87593 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -9,6 +9,7 @@ Standards-Version: 3.9.2
 Package: @(Package.replace('gazebo-','gazebo8-'))
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Conflicts: @(Package)
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.7.4

